src/GenericAccessRequestForm/Constants/FormSpecificConstants.tsx

```diff
@@ -1,20 +1,37 @@
+import React from 'react';
 import { workspaceRegistrationConfig } from '../../localconf';
 
+const defaultEmail = 'heal-support@datacommons.io';
```

Comment:
```suggestion
const defaultEmail = 'support@datacommons.io';
```
Let's use the generic Gen3 helpdesk email as the default

Reply:
Updated as requested